app/assets/javascripts/queue/queueController.js

```diff
@@ -81,8 +81,13 @@ angular.module('ocWebGui.queue', ['ocWebGui.queue.service', 'ui.router', 'ocWebG
     }
 
     function fetchContactStats() {
+      Settings.getOthers().then(function (others) {
+        vm.otherSettings = others;
+      });
+
       $http.get('contacts/stats.json').then(function (response) {
         var data = response.data;
+
         var callsValues = data.calls_by_hour
           .map(function (calls, hour) { return { hour: hour, calls: calls }; })
           .filter(function (item) { return item.hour >= 8 && item.hour <= 18; });
```

Comment:
Shouldn't this get the time interval from settings? Also `slice` would be work better here instead of `filter`.